vulcan/lib/server/workflows/umap.cwl

```diff
@@ -195,10 +200,10 @@ steps:
     run: scripts/calc_leiden.cwl
     label: 'Calculate Leiden clustering'
     in:
-      nn_anndata.h5ad: neighbors/nn_anndata.h5ad
+      nn_anndata.h5ad: calc_umap/umap_anndata.h5ad
```

Comment:
I've rewired the calc_leiden step to take in the output from calc_umap in order for us to have an object containing all pieces at the end.

This will be important for the now-planned, separate, sc Differential Expression workflow as that workflow will take in the downloaded .h5ad, and so will need 'leiden' stored within the object, but will also aim to provide a grid of umap plots as an output option.